### PR TITLE
add sharding-spring-boot-mybatis-example yml config file

### PR DESCRIPTION
**Pull request by xuelijun-dev**
add sharding-spring-boot-mybatis-example yml config file

Changes proposed in this pull request:
- examples/shardingsphere-jdbc-example/single-feature-example/sharding-example/sharding-spring-boot-mybatis-example/src/main/resources/application-sharding-tables.yml

### CODE REVIEW

**Comment by zjcnb** [comment]
@xuelijun-dev Hi, thank you for your pull request, your pull_request has ci error, and `sharding-spring-boot-mybatis-example` module it only support `xxx.properties`.

File: examples/shardingsphere-jdbc-example/single-feature-example/sharding-example/sharding-spring-boot-mybatis-example/src/main/resources/application-sharding-tables.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+spring:
+  shardingsphere:
```
Comment (soulasuna) [inline examples/shardingsphere-jdbc-example/single-feature-example/sharding-example/sharding-spring-boot-mybatis-example/src/main/resources/application-sharding-tables.yml:2]: Add license please

**Comment by terrymanu** [comment]
@xuelijun-dev The progress of this PR has not been upgraded for a long time. If there is no update, I will close it soon.

**Comment by terrymanu** [comment]
I just close this PR.

**Pull request closed without merging**